xla/tools/multihost_hlo_runner/README.md

```diff
@@ -43,3 +43,105 @@ In that case, a single GPU is necessary.
 -   Crashes:
     -   We may want to use `--dynamic_mode=off`.
     -   CUDA and Cudnn should be set up correctly.
+
```

Comment:
Maybe instead remove this file entirely and add it to tools.md, which is rendered at openxla.org, WDYT?

Reply:
Make sense to refactor.
But tools.md will end up much longer. Why not have tools.md have links a new page for that tools that will in the doc directory? I can do that next week.

Reply:
OK sure.